CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## v1.0.1
+
+- Fix Minitest with Knapsack #112 — @paulca
```

Comment:
Maybe use the PR description? `Fix project_dir issue expecting a string, not a Pathname`